pkgs/top-level/aliases.nix

```
@@ -874,6 +874,7 @@ mapAliases ({
   tokodon = plasma5Packages.tokodon;
   tor-browser-bundle-bin = tor-browser; # Added 2023-09-23
   transfig = fig2dev; # Added 2022-02-15
+  trust-dns = throw "trust-dns is rebranded as hickory-dns"; # Added 2023-10-20
```

```suggestion
  trust-dns = hickory-dns; # Added 2023-10-20
```
This should be a noop, right?